rotkehlchen/db/dbhandler.py

```diff
@@ -2338,6 +2350,101 @@ def get_exchange_credentials_extras(self, name: str, location: Location) -> Dict
 
         return extras
 
+    def add_binance_pairs(self, name: str, pairs: List[str], location: Location) -> None:
```

Comment:
Why is this function not used anywhere and why is only `set_binance_pairs()` used?

Reply:
Nice questions. This function only appends to the list of markets. For how we receive this information in the API it makes sense to use the `set_...` function. First I added this one and then when I had to call the API saw that the one needed was the other one. 

I didn't delete this one because thought that we may want at some point to autodetect markets and then this may be useful.